official/resnet/keras/keras_benchmark.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Executes Keras benchmarks and accuracy tests."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+from absl import flags
+from absl.testing import flagsaver
+import tensorflow as tf  # pylint: disable=g-bad-import-order
+
+
+DATA_DIR = '/data/cifar10_data/cifar-10-batches-bin'
+MIN_TOP_1_ACCURACY = 0.925
+MAX_TOP_1_ACCURACY = 0.938
+
+FLAGS = flags.FLAGS
+
+
+class KerasBenchmark(object):
+  """Base benchmark class with methods to simplify testing."""
+  local_flags = None
+
+  def __init__(self, output_dir=None, default_flags=None):
+    self.oss_report_object = None
+    self.output_dir = output_dir
+    self.default_flags = {}
+    if default_flags:
```

Comment:
http://pylint-messages.wikidot.com/messages:w0102

Reply:
Looking at it again, it is probably not a problem here because default_flags is immutable in `KerasBenchmark`. And if it is mutable, in the future we can do `self.default_flags = dict(default_flags)`, which I personally find more concise to read. Please feel free to choose the style you prefer.

Reply:
nice, I took a slightly different approach that I think you will find not too bad to read and a one liner.